addons/TBMod_zusatzStuff/eos/core/fn_core.sqf

```diff
@@ -101,20 +102,18 @@ if (getMarkerColor _mkr != "ColorBlack") then
 
     // SPAWN HOUSE PATROLS
     private _aGrp = [];
-    for "_counter" from 1 to _aGrps do
+    for "_counter" from 1 to (_aGrps + (_aGrpsIncrease * count allPlayers)) do
```

Comment:
Optimierung wäre statt 10 Mal einen `count allPlayers` zu machen, einfach oben:
`private _playerCount = count allPlayers;`

Reply:
Wird die funktion mehrfach aufgerufen? oder Teile, das habe ich noch nicht ganz verstanden

Reply:
Hier wird sie nur einmal ausgeführt, aber in Zeile:
https://github.com/shukari/TBMod/pull/33/files#diff-6890ac9586ee133c4a34515773786979R114
https://github.com/shukari/TBMod/pull/33/files#diff-6890ac9586ee133c4a34515773786979R134
https://github.com/shukari/TBMod/pull/33/files#diff-6890ac9586ee133c4a34515773786979R142
usw. jeweils nochmal unnötiger weise, weil sich die Anzahl nicht in 0.03ms verändert bzw dann unwichtig ist ;D